packages/components/datepicker/README.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Datepicker'
+status: 'alpha'
```

Comment:
Is the API stable? Then it's beta, no? 😁

Reply:
don't even start 😄